### ISSUE TITLE
Blank page will somehow show up when webview tag loads a url at first time.

### ISSUE THREAD

**Issue by qazbnm456** [qazbnm456/lulumi-browser#3]
## Electron Version: 1.6.1-beta

As titled, and the screenshot attached below:

<img src="http://i.imgur.com/R31E6Fv.png" />

This problem happened to **lulumi-browser** when I upgraded electron to version 1.6.1-beta.

**Comment by tonyganch** [comment]
Hi @qazbnm456, could you please upload screenshot one more time? It doesn't load now for some reason.

**Comment by qazbnm456** [comment]
@tonyganch Sure!

![2017-03-01 10 30 35](https://cloud.githubusercontent.com/assets/6451933/23443513/33e5e65e-fe6a-11e6-8be9-0d793f994aed.png)

**Comment by qazbnm456** [comment]
I think the problem has something to do with Vue.js, especially **Vuex**. The problem could happen if I assigned that initial page object in an early time (maybe even before render processing of Electron). 

From the [fix](https://github.com/qazbnm456/lulumi-browser/commit/238cb9084f5af8bc55f2bb66d965ce16d83e610f), I moved the assignment statement to **beforeMount** event (inside `this.$store.dispatch('createTab')`), and the problem has vanished away.

But, still be eager to create a repro to see what actually happens!
